### ISSUE TITLE
Get url of thumbnail image from code tab

### ISSUE THREAD

**Issue by Alex360hd** [toughdeveloper/oc-imageresizer-plugin#27]
Hello,

I'm trying to get a thumbnail image url in the code tab of my page, in order to generate an url for og:image meta tag.

I can create an image object using this code :

    use ToughDeveloper\ImageResizer\Classes\Image;
    $image = new Image($record->images[0]->path);
    $image->resize(150, 200, ['mode' => 'crop']);

But then, I don't know how to retrieve the thumbnail url as a string.

Could you point me the best way to get it ?

Thanks

Alex

**Comment by toughdeveloper** [comment]
@Alex360hd - Hey,

This should get you the path (the parameter indicates you want the public url rather than the server path):

```
$image->resize(150, 200, ['mode' => 'crop']);
$thumb = $image->getCachedImagePath(true); 
<meta property="og:image" content="<?=$thumb?>" />
```

Although the plugin utilises the `__toString()` magic method. So, you can shorten this to:

```
$thumb = $image->resize(150, 200, ['mode' => 'crop']);
<meta property="og:image" content="<?=$thumb?>" />
```

Hope this helps, let me know if you have any other problems.

Matt

**Comment by Alex360hd** [comment]
Thanks, I couldn't find the getCachedImagePath function, it works !
